### ISSUE TITLE
after setting entry id, entry becomes not dragable

### ISSUE THREAD

**Issue by tomas-puplausks** [stefanuebe/vaadin_fullcalendar#64]
If you set entry id for example to '5ee48b8addb3ad2d823e151e' - entry becomes not drag-able. However if you leave it empty, id will be auto-generated and it will be drag-able.

Tested on 2.3.2 fullcalendar2-scheduler/fullcalendar2 and on snapshot.

As workaround i set my id to entry description.

**Comment by stefanuebe** [comment]
Hi, I'm currently catched in a big project. I try to check out this issue as soon as possible. Sorry for the inconvenience.

**Comment by aetasoul** [comment]
Hi @tomas-puplausks I can confirm the issue, the problem is that when you set the id the css classes `fc-draggable fc-resizable` are removed from the event.

![image](https://user-images.githubusercontent.com/15131692/99507860-3534b280-2984-11eb-9589-c2da2589f7d9.png)

Classes in event **Multi 10**:
```
fc-day-grid-event fc-h-event fc-event fc-start fc-end fc-draggable fc-resizable
```
Classes in event **Multi 1**:
```
fc-day-grid-event fc-h-event fc-event fc-start fc-end
```

I'm investigating on it

**Comment by tomas-puplausks** [comment]
Hi, thank you very much.

**Comment by aetasoul** [comment]
I think the problem is [here](https://github.com/stefanuebe/vaadin_fullcalendar/blob/master/addon/src/main/java/org/vaadin/stefan/fullcalendar/Entry.java#L180), this code just set the id of the event, and the editable property is false by default. Take a look [here](https://github.com/stefanuebe/vaadin_fullcalendar/blob/master/addon/src/main/java/org/vaadin/stefan/fullcalendar/Entry.java#L167) as you can see after generating a random id for the event the editable property is set to true, so to solve your problem:

*  Create a new event: `Entry entry = new Entry("5ee48b8addb3ad2d823e151e");`
*  Set te editable property: `entry.setEditable(true);`

I'll open a pull request to solve the issue changing [this class](https://github.com/stefanuebe/vaadin_fullcalendar/blob/master/addon/src/main/java/org/vaadin/stefan/fullcalendar/Entry.java#L180) as follow:

```java
public Entry(String id) {
    this.id = id != null ? id : UUID.randomUUID().toString();
    this.editable = true;
}
```

By default setting the ID manually the events are not editable, this could be a breaking changes for someone, I let @stefanuebe to decide if implement the pull request or not.

**Comment by aetasoul** [comment]
The problem is solved with the merge done today. I close the issue, we will release the fix on the Vaadin Directory as soon as possible.

**Comment by aetasoul** [comment]
Hi @tomas-puplausks We have finally released the version 2.4.0 on the [Vaadin Directory](https://vaadin.com/directory/component/full-calendar-4-scheduler-web-component/overview)

Update your FC version and if you experience any other problem please report it.
